.github/CONTRIBUTING.md

```diff
@@ -1,66 +1,57 @@
+* check for [JavaScript errors with your browser's console](https://kb.yoast.com/kb/how-to-find-javascript-errors-with-your-browsers-console/). Please report your findings in the issue.
+* add as much information as possible. For example: add screenshots, relevant links, step by step guides etc.
 
-1. What you were trying to achieve.
-2. What you were expecting to happen.
-3. What actually happened, illustrated with screenshots if possible.
+## I have a feature request
+Before opening a new issue, please:
+* search for duplicate issues to prevent opening a duplicate feature request. If there is already an open existing request, please leave a comment there.
+* add as much information as possible. For example: give us a clear explanation of why you think the feature request is something we should consider for the Yoast SEO plugins.
 
-Your bug report should also contain your WordPress version and if there are any errors, the _exact_ error text, including line numbers.
-
-### Blank / white screen
-If you're getting a blank screen and you report just that, we can do _absolutely_ nothing. By default, your WordPress install suppresses all errors, to prevent information leaks, but we need those errors to be able to help you. If you apply the small piece of code in [this post on WP_DEBUG](https://yoast.com/wordpress-debug/) to your site, you should be able to open the URL that gave you a white screen, append `?debug=debug` to the URL and get the actual error.
-
-That error will help us, without that error, we're completely in the dark about your white page problem...
-
-### Interface errors
-If you're reporting a bug about specific interface elements not working as expected, there's probably an error showing in your browsers JavaScript console. Please open your browsers console and copy the exact error showing there, or make a screenshot. If you don't know how to open your browsers console, here is info for [Chrome](https://developer.chrome.com/devtools/docs/console) and [Firefox](https://developer.mozilla.org/en/docs/Tools/Web_Console). For IE, some Googling will help but it changes with every version.
-
-<a name="security"></a>
-#Security issues
-Please do _not_ report security issues here, but email them to security at yoast dot com so we can deal with them securely and quickly.
-
-<a name="contribute"></a>
-#Contribute to Yoast SEO
-
-Community made patches, localisations, bug reports and contributions are very welcome and help make Yoast SEO remains the #1 SEO plugin for WordPress.
+## I want to create a patch
+Community made patches, localizations, bug reports and contributions are very welcome and help Yoast SEO remain the #1 SEO plugin for WordPress.
 
 When contributing please ensure you follow the guidelines below so that we can keep on top of things.
 
-## Getting Started
+#### Getting Started
+If there is no ticket for your issue, submit it first, following the guidelines above.
```

Comment:
If you cannot find anything which resembles your problem, please [create a new issue](https://github.com/Yoast/wordpress-seo/issues/new).